packages/terra-clinical-data-grid/package.json

```diff
@@ -37,13 +37,13 @@
     "prop-types": "^15.5.8",
     "react-draggable": "^3.0.5",
     "resize-observer-polyfill": "^1.4.1",
-    "terra-button": "^2.0.0",
+    "terra-button": "^3.0.0",
     "terra-clinical-item-view": "^2.16.0",
-    "terra-content-container": "^2.0.0",
-    "terra-doc-template": "^1.0.0",
-    "terra-icon": "^2.0.0",
-    "terra-mixins": "^1.0.0",
-    "terra-overlay": "^2.0.0"
+    "terra-content-container": "^2.0.1",
+    "terra-doc-template": "^2.0.0",
+    "terra-icon": "^2.6.0",
+    "terra-mixins": "^1.21.0",
```

Comment:
uh oh, looks like we lost some of the changes you originally made

Reply:
Should I change those back? The new changes from master don't throw the same hoist warnings.

Reply:
yeah, we should match all dependencies to a major version `^2.0.0` or `^3.0.0` etc

Reply:
it will make it easier to manage if/when we add new components to the repo

Reply:
Packages updated!